package.json

```diff
@@ -53,6 +55,9 @@
     "overrides": {
       "@gadgetinc/api-client-core": "workspace:*",
       "@gadgetinc/react": "workspace:*"
+    },
+    "patchedDependencies": {
+      "@n1ru4l/push-pull-async-iterable-iterator@3.2.0": "patches/@n1ru4l__push-pull-async-iterable-iterator@3.2.0.patch"
```

Comment:
Fun fact: the types are wrong in this package and it causes issues in ours in attw. This fixes it locally so our tests pass.